### ISSUE TITLE
Unclear setup of source maps

### ISSUE THREAD

**Issue by Arhane** [postcss/postcss-loader#318]
I have been trying to set up sourceMaps with the last version of postcss-loader. And it got really messy.

I set up `sourceMap=true` in webpack config, and I set up `map="inline"` in postcss.config.js .

If I remove  `sourceMap=true`, source maps don't work. If I remove `map="inline"`, source maps don't work correctly. My main concern is that we configure the source map usage in two different places and that doesn't get along with the [SRP](https://en.wikipedia.org/wiki/Single_responsibility_principle).

I suggest to remove sourceMap from webpack config, that way it will only stay in postcss config  with other plugins and options.

P.S. Is the environment (npm, node, etc...) important here?

**Comment by michael-ciniawsky** [comment]
You can't configure source maps via `postcss.config.js` (remove it anything related to source maps entirely from the config file please) when using postcss-loader (webpack), since webpack sets [strict] requirements about the source map generation (format). The 90%  correct way to go is simply using `sourceMap: true` in `webpack.config.js`'s `postcss-loader`options.`sourceMap: inline` can be used with `style-loader` && `postcss-loader`  (standalone, no `css-loader`) so the source map is inlined in the CSS source as an annotation comment `/*# sourceMappingURL= ... */`  which loads faster in development.

TL:DR remove all source map related props from `postcss.config.js` && use `sourceMap: true` in `webpack.config.js` `postcss-loader` options

Feel free to reopen if still regressions
